src/Symfony/Component/HttpKernel/EventListener/ProfilerListener.php

```diff
@@ -79,6 +81,10 @@ public function onKernelResponse(ResponseEvent $event)
         }
 
         $request = $event->getRequest();
+        if (null !== $this->collectParameter && null !== $collectParameterValue = $request->get($this->collectParameter)) {
```

Comment:
I'm wondering if we really want to support passing it as attribute or server parameter. Isn't the query param enough?

Reply:
I was doubtful at first but this enables a whole class of new usage, such as enabling (or disabling) the profiler only for some routes (using defaults). Also, it's in sync with what we do in other parts of SF, and as it's a dev-only feature, this can cause any harm.

Reply:
fair enough